src/hooks/useCSVListExport/downloadCSV.ts

```diff
@@ -1,15 +1,14 @@
-import { KyInstance } from 'ky/distribution/types/ky';
-// @ts-ignore:next-line
 import { downloadBase64 } from '@folio/stripes-acq-components';
+import { useOkapiKy } from '@folio/stripes/core';
```

Comment:
Lingering headache from a few weeks back, where `ky` versions got scrambled and caused weird missing types in some platform builds. We fixed this, but my IDE is still mad, and if we're not using `ky` directly (and instead its return from stripes-core), we should be using the type from stripes-core, too.